pkgs/servers/corosync/default.nix

```diff
@@ -9,11 +9,13 @@
 with stdenv.lib;
 
 stdenv.mkDerivation rec {
-  name = "corosync-2.4.3";
+  pname = "corosync";
+  version = "2.4.5";
+  name = "${pname}-${version}";
```

Comment:
Not needed.

Reply:
You could argue that it won't hurt if it's here and used in `src`, but we'd like to get rid of direct usages of `name` in derivations due to evaluation of repology (can't find right now the links to discussions..).